.changes/unreleased/polaris-TINY-11650-2025-03-10.yaml

```diff
@@ -0,0 +1,6 @@
+project: polaris
+kind: Changed
+body: Changed word-capture regex for abbreviations
```

Comment:
This doesn't say anything about why. It's a fix, not a change just because we felt like changing it.

- [ ] reword as fix